.idea/.gitignore

```diff
@@ -1,3 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
```

Comment:
Don't delete this file. You can find an example of it here https://gist.github.com/dedunumax/54e82214715e35439227
Files in .idea and .iml files are not committed usually